step-capstone/src/App.js

```diff
@@ -1,12 +1,46 @@
 import React from 'react';
 import './styles/App.css';
 import TravelObject from './components/TravelObject'
+import { Box, Grid, Typography } from '@material-ui/core'
 
 class App extends React.Component {
-  render () {
-    return(
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: []
+    }
+  }
+
+  fetchData() {
+    return [
```

Comment:
Could this be wired up to the trip dataset you define below instead? That seems like better test data.

Reply:
Oh this is really old code, sorry. I've moved everything to Trip.js instead of App but I didn't push App